docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -131,38 +126,32 @@ done
 . Get the count of not-OK source databases:
 +
 ----
-oc rsh mariadb-copy-data mysql -h "${SOURCE_MARIADB_IP}" -uroot -p"${SOURCE_DB_ROOT_PASSWORD}" -e "SHOW databases;"
+$ oc rsh mariadb-copy-data mysql -h "${SOURCE_MARIADB_IP}" -uroot -p"${SOURCE_DB_ROOT_PASSWORD}" -e "SHOW databases;"
 ----
 
-. Run mysqlcheck on the original DB to look for things that are not OK:
+. Run `mysqlcheck` on the original database to check for things that are not OK:
```

Comment:
I'm not sure (never saw it happen myself). We'll probably want to tweak the snippets below so that error/success conditions are more obvious. There is a bug reported for that: https://issues.redhat.com/browse/OSPRH-7859

Reply:
@bogdando Could we reword it to: "Run `mysqlcheck` on the original database to check for errors"?